src/components/HorizontalStep/HorizontalStep.module.css

```diff
@@ -0,0 +1,59 @@
+/*------------------------------------*\
+    # HORIZONTAL STEP
+\*------------------------------------*/
+
+/**
+ * Horizontal Step displays the step text and associated decorative icon.
+ */
+.horizontal-step {
+  display: flex;
+  align-items: center;
+  gap: var(--eds-size-1);
+}
+
+/**
+ * Horizontal step variants
+ */
+.horizontal-step--complete {
+  color: var(--eds-theme-color-text-neutral-subtle);
+}
+.horizontal-step--active {
+  color: var(--eds-theme-color-text-neutral-strong);
+}
+.horizontal-step--incomplete {
+  color: var(--eds-theme-color-text-neutral-subtle);
+}
+
+/**
+ * Horizontal Step Complete
+ */
+.horizontal-step__complete-icon {
+  fill: var(--eds-theme-color-icon-utility-success);
+}
+
+/**
+ * Horizontal Step Incomplete
+ */
+.horizontal-step__incomplete-icon {
+  margin-left: var(--eds-size-1);
+  margin-right: var(--eds-size-1);
```

Comment:
These left and right margins look like they're causing the gaps to be too wide

Reply:
Good callout, this was done intentionally to keep the `incomplete-icon` width the same as the other icons, so there's no jitter when navigating. Of course would have to pass it by design first, but was waiting until the actual pilot launch for that.

Without the margins, jitter:

https://user-images.githubusercontent.com/86632227/165593826-476df1dd-c43f-4129-b0f7-b18503e31547.mov

Reply:
Confirmed with design that we don't want jitter and would rather have extra spacing